TYPO3.Neos/Migrations/Mysql/Version20151223125909.php

```diff
@@ -0,0 +1,50 @@
+<?php
+namespace TYPO3\Flow\Persistence\Doctrine\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration,
+    Doctrine\DBAL\Schema\Schema;
+use TYPO3\Neos\Utility\User as UserUtility;
+
+/**
+ * Set the Workspace "owner" field for all personal workspaces with special characters in the username
+ */
+class Version20151223125909 extends AbstractMigration
+{
+
+    /**
+     * @param Schema $schema
+     * @return void
+     */
+    public function up(Schema $schema)
+    {
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() != "mysql");
+
+        $workspacesWithoutOwnerQuery = $this->connection->executeQuery('SELECT name FROM typo3_typo3cr_domain_model_workspace t0 WHERE t0.name LIKE \'user-%\' AND t0.owner IS NULL');
```

Comment:
Narf, but it worked on my machine :( Will have a look, your explanation makes sense. So probably should be adjusted once more.